__main__ - training_phase(line:1376) - ERROR: The file trace.csv does not exist.
__main__ - training_phase(line:1377) - @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
apscheduler.executors.default - run_job(line:131) - Job "optimizer_entrypoint (trigger: interval[0:00:01], next run at: 2024-04-05 19:09:53 UTC)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/app/global_controller.py", line 1006, in optimizer_entrypoint
    assert False # TODO: not implemented yet
AssertionError
apscheduler.executors.default - run_job(line:131) - Job "optimizer_entrypoint (trigger: interval[0:00:01], next run at: 2024-04-05 19:10:35 UTC)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/app/global_controller.py", line 953, in optimizer_entrypoint
    weight = flow / total
ZeroDivisionError: division by zero
apscheduler.executors.default - run_job(line:131) - Job "optimizer_entrypoint (trigger: interval[0:00:01], next run at: 2024-04-05 19:10:36 UTC)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/app/global_controller.py", line 953, in optimizer_entrypoint
    weight = flow / total
ZeroDivisionError: division by zero
